services/webhook/deliver.go

```diff
@@ -78,7 +78,7 @@ func Deliver(t *models.HookTask) error {
 			return err
 		}
 	case http.MethodPut:
-		switch t.Typ {
+		switch strings.TrimSpace(t.Typ) {
```

Comment:
Unnecessary

Reply:
This is for wrong content on the database because the previous PR.

Reply:
Please have migration fix the data in database instead of writing code that tries to work around this. In future, we won't remember of such edge case and it might cause unnecessary issues for users.

Reply:
Will do that.